utils/generate_report.py

```
@@ -75,35 +77,34 @@ def generate_full_process_report(
+        template_path = os.path.join(template_dir, "process_report_template.html")
+        logger.debug(f"Checking for template file: {template_path}", indent=2)
 
         if not os.path.exists(template_path):
-            raise FileNotFoundError(
-                f"Template file not found at: {template_path}. "
-                f"Please ensure the template exists in the {template_dir_resolved} directory "
-                f"or check the 'template.templates_dir' setting in your configuration."
-            )
+            logger.error("Template file not found at: {template_path}", indent=3)
+            logger.error(f"Please ensure the template exists in the {template_dir} directory", indent=3)
+            logger.error(f"or check the 'template.templates_dir' setting in your configuration.", indent=3)
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)